modules/assim.batch/R/pda.emulator.R

```diff
@@ -194,13 +321,33 @@ pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NU
       
       ## Convert probabilities back to parameter values
       for(i in 1:n.param) {
-        m[,i] <- eval(prior.fn$qprior[prior.ind][[i]], list(p=mcmc.out[[c]][,i]))
+        m[,i] <- eval(prior.fn$qprior[prior.ind][[i]], list(p=mcmc.out[[c]]$mcmc[,i]))
       }
     }
-      colnames(m) <- pname[prior.ind]
-      mcmc.out[[c]] <- m
+    colnames(m) <- pname[prior.ind]
+    mcmc.list.tmp[[c]] <- m
+    
+    jvar.list[[c]] <- sqrt(mcmc.out[[c]]$jump@history[nrow(mcmc.out[[c]]$jump@history),])
```

Comment:
jump was my first foray into writing a class years ago and I'm sure it could be implemented better. But I'd leave it well enough alone for this release at least